<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Beautifier::Internal::GeneralOptionsPage</class>
 <widget class="QWidget" name="Beautifier::Internal::GeneralOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="formatOnSave">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Automatic Formatting on File Save</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="autoFormat">
        <property name="text">
         <string>Enable auto format on file save</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Tool:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="autoFormatTool">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Restrict to MIME types:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="autoFormatMime">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="autoFormatOnlyCurrentProject">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Restrict to files contained in the current project</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>autoFormat</sender>
   <signal>toggled(bool)</signal>
   <receiver>autoFormatTool</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoFormat</sender>
   <signal>toggled(bool)</signal>
   <receiver>autoFormatMime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoFormat</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoFormat</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoFormat</sender>
   <signal>toggled(bool)</signal>
   <receiver>autoFormatOnlyCurrentProject</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
